frontend/apps/app/app/(app)/app/projects/[projectId]/ref/[branchOrCommit]/schema/[...schemaFilePath]/page.tsx

```diff
@@ -137,33 +131,25 @@ export default async function Page({
 
     setPrismWasmUrl(path.resolve(process.cwd(), 'prism.wasm'))
 
-    let format: SupportedFormat | undefined
-    const searchParams = await _searchParams
-
-    if (v.is(searchParamsSchema, searchParams)) {
-      format = searchParams.format
-    }
-    if (!format) {
-      format = detectFormat(filePath)
-    }
-
-    if (!format) {
+    if (!gitHubSchemaFilePath?.format) {
```

Comment:
ask:

Is it possible for this to be missing? I thought a persisted GitHubSchemaFilePath table's record has always format value(non-null).